app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt

```diff
@@ -419,6 +419,10 @@ class ProfileProgressFragmentTest {
 
```

Comment:
test `testProfileProgressFragment_twoPartialStoryProgress_completedStoriesDescriptionIsCorrect` is failig on espresso

Reply:
```
java.lang.NullPointerException
at org.oppia.android.app.recyclerview.RecyclerViewMatcher$Companion$atPositionOnView$1.matchesSafely(RecyclerViewMatcher.kt:53)
at org.oppia.android.app.recyclerview.RecyclerViewMatcher$Companion$atPositionOnView$1.matchesSafely(RecyclerViewMatcher.kt:29)
```

Reply:
Fixed.